esi_leap/common/policy.py

```diff
@@ -47,13 +56,17 @@
         [{'path': '/leases', 'method': 'POST'}]),
     policy.DocumentedRuleDefault(
         'esi_leap:lease:get',
-        'rule:is_admin or rule:is_lessee or rule:is_owner',
+        'rule:is_admin or rule:is_lease_owner or rule:is_lease_lessee',
```

Comment:
When retrieving a lease, what's the difference between 'rule:is_lease_owner or rule:is_lease_lessee' and 'rule:is_lessee or rule:is_owner'? I think the lessee or owner can only get the lease they own.

Reply:
Good question! These policy rules are defined near the top of the file: is_lessee and is_owner means that the user has the lessee or owner role, while is_lease_owner/is_lease_lessee means that the user's project_id matches the owner or lessee of the lease.

Previously the code that ensured that a lessee or owner can only get the lease they own was very specifically coded with several if statements in the controller. This change pushes that check to oslo.policy.

Reply:
Gocha!